src/Bundle/Tests/Unit/FieldTypes/TwigFieldTypeTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\GridBundle\Tests\Unit\FieldTypes;
+
+use PHPUnit\Framework\TestCase;
+use Sylius\Bundle\GridBundle\FieldTypes\TwigFieldType;
+use Sylius\Component\Grid\DataExtractor\DataExtractorInterface;
+use Sylius\Component\Grid\Definition\Field;
+use Sylius\Component\Grid\FieldTypes\FieldTypeInterface;
+use Twig\Environment;
+
+final class TwigFieldTypeTest extends TestCase
+{
+    private TwigFieldType $fieldType;
+
+    private DataExtractorInterface $dataExtractor;
+
+    private Environment $twig;
+
+    protected function setUp(): void
+    {
+        $this->dataExtractor = $this->createMock(DataExtractorInterface::class);
+        $this->twig = $this->createMock(Environment::class);
+        $this->fieldType = new TwigFieldType($this->dataExtractor, $this->twig);
+    }
+
+    public function testItIsAGridFieldType(): void
```

Comment:
According to naming scheme at  https://github.com/Sylius/SyliusGridBundle/pull/399/files `it` should be dropped. 

```suggestion
    public function testIsAGridFieldType(): void
```